game/database/schema/collection.lua

```diff
@@ -1,6 +1,7 @@
 
 return {
   { id = 'name', name = "Full Name", type = "string" },
+  { id = 'image', name = "Image", type = 'enum', options = 'resources.texture' },
```

Comment:
You don't use this field, right?

Reply:
I use it to draw the pack icon

Reply:
Oh, right, Nevermind.

Reply:
Line 189 of file packlist.lua

Reply:
I'll rebase the branch as soon as possible ^^

Reply:
Yes, I meant to say it was me who got it wrong. My bad.

Reply:
No prob ^^